### DISCUSSION TITLE
Multiples regions through Docker Windows 10.

### DISCUSSION THREAD

**Discussion by 8m0** [mediagis/nominatim-docker#244] [Q&A]
Hello,

I have read the guide https://nominatim.org/release-docs/develop/admin/Advanced-Installations/#importing-multiple-regions, and don't understand how do I implement it through Docker on Windows 10. Is there some more detailed guide I could miss?

**Comment by philipkozeny** [comment]
Hi,

this feature is not supported from the docker image right away right now. There is an open PR (https://github.com/mediagis/nominatim-docker/pull/186) which needs major refactoring. Right now I would advise you to use the region that has all the data you need, sorry.
